packages/shared/src/hooks/useActions.ts

```diff
@@ -21,7 +21,7 @@ export const useActions = (): UseActions => {
   const { data: actions } = useQuery(
     generateQueryKey(RequestKey.Actions, user),
     getUserActions,
-    { enabled: !!user },
+    { enabled: !!user?.id },
```

Comment:
This change shouldn't be needed as it is coming from the context, and if it is anonymous user, we return null for user.

Reply:
I just thought no point in doing a separate API request for anonymous since it will just return nothing. This way it should be only one once we know user is logged in and actually capable of completing actions.

Reply:
I mean on an anonymous user this won't make a request still because the `user` object is null if it is anonymous.

Reply:
In some cases I was getting `{ name: 'anonymous' }` or something like that, tried it now and I can not get it anymore. You might know?

Reply:
Oh, interesting. I'll be sure to watch out if I encounter any. Just FYI, if ever you need the anonymous data (like tracking id) there should be an `anonymous` property from `AuthContext` 🚀